### ISSUE TITLE
Data availability?

### ISSUE THREAD

**Issue by vsoch** [snakemake-workflows/rna-seq-star-deseq2#56]
Hi! I'm using this as a test workflow for an orchestration tool, and I'm not familiar with the workflow itself and am looking for some dummy data so this doesn't happen:

```
Building DAG of jobs...
MissingInputException in rule align  in line 1 of https://github.com/snakemake-workflows/rna-seq-star-deseq2/raw/v1.2.0/workflow/rules/align.smk:
Missing input files for rule align:
    output: results/star/A-lane1/Aligned.sortedByCoord.out.bam, results/star/A-lane1/ReadsPerGene.out.tab
    wildcards: sample=A, unit=lane1
    affected files:
        A.2.fq.gz
        A.1.fq.gz
```
Is this readily available, and if not, is there another workflow in the catalog with data that is (maybe beyond the basic Snakemake getting started workflow?) Thank you!

**Comment by dlaehnemann** [comment]
In the https://github.com/snakemake-workflows, we usually try to have basic test data in the `.test/` directory in the main folder of the repo. Here, this is in [the folder `.test/ngs-test-data/reads/`](https://github.com/snakemake-workflows/rna-seq-star-deseq2/tree/v1.2.0/.test/ngs-test-data/reads).

These are then used during GitHub Actions continuous integration testing. For this workflow, see here for the tests that are run:

https://github.com/snakemake-workflows/rna-seq-star-deseq2/blob/dfd0b9cb8c1ef8607505aef8baf885632227e8ac/.github/workflows/main.yml#L35-L56

Does this help?

**Comment by vsoch** [comment]
Yes that's perfect! I totally glossed over the test directory, thanks for the helpful reminder.
